be/src/runtime/exec_env.cpp

```diff
@@ -473,6 +483,7 @@ void ExecEnv::_destroy() {
     SAFE_DELETE(_connector_scan_executor_without_workgroup);
     SAFE_DELETE(_connector_scan_executor_with_workgroup);
     SAFE_DELETE(_thread_pool);
+    SAFE_DELETE(_streaming_load_thread_pool);
```

Comment:
should before _fragment_mgr?

Reply:
should stop the fragment first. `ConnectorScanNode::close` will close and wait the scanner to finish, and after that the _streaming_load_thread_pool will not be used, and can be deleted safely